README.md

```diff
@@ -561,9 +573,30 @@ The `access_token` granted by the Credential Issuer contains the following [JWT
 | `c_nonce`        | A nonce to be used in the [Proof JWT](#proofjwt-claims)              | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-6.2-4.1) |          |
 | `c_nonce_expiry` | Expiry time of the c_nonce                                           | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-6.2-4.1) |          |
 
-### 3. Issuance Endpoints
+#### Token Error Response
+
+The authorization server responds with an HTTP 400 (Bad Request), and an error object in the response body.
 
-Clients interface with the following endpoints as a means of acquiring the credential.
+| Field               | Description                                                | Required | References                                                           | Comments |
+| :------------------ | :--------------------------------------------------------- | :------- | :------------------------------------------------------------------- | :------- |
+| `error`             | A string error code from below list                        | y        | [RFC6749](https://datatracker.ietf.org/doc/html/rfc6749#section-5.2) |          |
+| `error_description` | A human-readable description of the error                  | n        | [RFC6749](https://datatracker.ietf.org/doc/html/rfc6749#section-5.2) |          |
+| `error_uri`         | A URI that provides additional information about the error | n        | [RFC6749](https://datatracker.ietf.org/doc/html/rfc6749#section-5.2) |          |
+
+##### Error Codes
+
+Error codes are defined in [OID4VCI Token Error Response](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-6.3).
+
+- `invalid_request`
+- `invalid_grant`
+- `invalid_client`
+- `unauthorized_client`
+- `unsupported_grant_type`
+
+### Credential Endpoint
+
+This endpoint issues a Credential (or a Transaction ID) upon presentation of a valid Access Token.
+The URL is the `credential_endpoint` field of the [Credential Issuer Metadata](#credential-issuer-metadata).
 
 ```mermaid
 sequenceDiagram
```

Comment:
Thoughts on adding [an `alt`/`else` block](https://mermaid.js.org/syntax/sequenceDiagram.html#alt) for sync vs async flows? My general thinking is... we want to convey that the _Credential Endpoint_ is hit first, but it doesn't necessitate the _Deferred Credential Flow_.

Reply:
Niiice, done 👍